.github/workflows/ci.yml

```
@@ -63,25 +65,25 @@ jobs:
     steps:
 
     - name: Checkout Code
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
+
+    - name: Create Build Environment
+      run: cmake -E make_directory ${{runner.workspace}}/build
 
     - name: Configure
-      working-directory: test
-      run: cmake -S . -B build ${{ matrix.cmake_opts }}
+      working-directory: ${{runner.workspace}}/build
+      shell: bash
+      run: cmake -DARGPARSE_BUILD_SAMPLES=ON -DCMAKE_BUILD_TYPE=Debug ${{ matrix.cmake_opts }} $GITHUB_WORKSPACE
       env:
         CC: ${{ matrix.c_compiler }}
         CXX: ${{ matrix.cxx_compiler }}
 
     - name: Build for ${{ matrix.os }} with ${{ matrix.compiler }}
-      working-directory: test
-      run: cmake --build build
+      working-directory: ${{runner.workspace}}/build
+      run: cmake --build . --config Debug --verbose
```

No need to set the `working-directory`, we can just use this:
```suggestion
      run: cmake --build build --config Debug --verbose
```